### PR TITLE
PMM-2832 fix documenting how to change pmm-server options in docker

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-251/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-251/

File: doc/source/glossary.option.rst [hunk 1]
```diff
@@ -7,6 +7,9 @@
 This glossary contains the addtional parameters that you may pass when starting
 |pmm-server|.
 
+Passing options to *PMM Server* on the docker initial start up
```
Comment (michaelcoburn) [inline doc/source/glossary.option.rst]:
```suggestion
Passing options to PMM Server docker at first run```

File: doc/source/glossary.option.rst [hunk 2]
```diff
@@ -16,6 +19,47 @@ command. |tip.run-this.root|.
 
 .. include:: .res/code/docker.run.server-user.example.txt
 
+Passing options to *PMM Server* already deployed via docker
```
Comment (michaelcoburn) [inline doc/source/glossary.option.rst]:
```suggestion
Passing options to *PMM Server* for an already deployed docker instance
```

File: doc/source/glossary.option.rst [hunk 3]
```diff
@@ -7,6 +7,9 @@
 This glossary contains the addtional parameters that you may pass when starting
 |pmm-server|.
 
+Passing options to *PMM Server* on the docker initial start up
+================================================================================
+
 If you use |docker| to run the server, use the :option:`-e` flag followed by the
```
Comment (michaelcoburn) [inline doc/source/glossary.option.rst]:
```suggestion
|docker| allows configuration options to be passed using the flag :option:`-e` flag followed by the option you would like to set.
```

File: doc/source/glossary.option.rst [hunk 4]
```diff
@@ -7,6 +7,9 @@
...
 parameter. Use this flag in front of each parameter that you pass to
```
Comment (michaelcoburn) [inline doc/source/glossary.option.rst]:
```suggestion
```

File: doc/source/glossary.option.rst [hunk 5]
```diff
@@ -7,6 +7,9 @@
...
 |pmm-server|.
```
Comment (michaelcoburn) [inline doc/source/glossary.option.rst]:
```suggestion
```

File: doc/source/glossary.option.rst [hunk 6]
```diff
@@ -16,6 +19,47 @@ command. |tip.run-this.root|.
...
+doesn't support changing environment variables. Therefore if you want container
+with different properties,  you should run a new container instead.
+
+1. Stop and Rename the old container, just in case you want to go back::
+
+   docker stop pmm-server
+   docker rename pmm-server pmm-server-old
+
+2. Refresh the container with the latest
+   version::
```
Comment (michaelcoburn) [inline doc/source/glossary.option.rst]:
```suggestion
```

File: doc/source/glossary.option.rst [hunk 7]
```diff
@@ -16,6 +19,47 @@ command. |tip.run-this.root|.
...
+
+      docker pull percona/pmm-server:latest
+
+   .. warning:: When you destroy and recreate the container, all the
+      updates you have done through PMM Web interface will be lost. What’s more,
+      the software version will be reset to the one in the Docker image. Running
+      an old PMM version with a data volume modified by a new PMM version may
+      cause unpredictable results. This could include data loss.
+
+4. Run the container with the new settings. For example, changing
```
Comment (michaelcoburn) [inline doc/source/glossary.option.rst]:
```suggestion
4. Start the container with the new settings. For example, changing
```

File: doc/source/glossary.option.rst [hunk 8]
```diff
@@ -16,6 +19,47 @@ command. |tip.run-this.root|.
...
+
+      docker run -d \
+        -p 80:80 \
+        --volumes-from pmm-data \
+        --name pmm-server \
+        --restart always \
+        -e METRICS_RESOLUTION=5s \
+        percona/pmm-server:latest
+
+5. After you’re happy with your new container deployment you can remove the old
```
Comment (michaelcoburn) [inline doc/source/glossary.option.rst]:
```suggestion
5. Once you're satisfied with the new container deployment options and you don't plan to revert, you can remove the old
```

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-251/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-251/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-251/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-251/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-251/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-251/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-251/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-251/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-251/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-251/

**Pull request merged**